tox.ini

```diff
@@ -37,7 +37,8 @@ recreate = False
 commands =
     pip install -U https://github.com/celery/celery/zipball/master#egg=celery
     pip install -U https://github.com/celery/kombu/zipball/master#egg=kombu
-    py.test -xv
+    py.test -xv --ignore=t/unit/test_models.py
```

Comment:
Why are you ignoring the test you added?

Reply:
The new tests are run with the line below: `py.test -xv -k 'ModelMigrationTests'`. It is necessary, to run the new tests separately, because they override settings used in models and it is therefore necessary, to do some `six.moves.reload_module` calls in the tests, which breaks subsequent test runs, that rely on the same models.